app/components/marketing/events-section.tsx

```diff
@@ -44,18 +45,23 @@ export function EventCard({ event, index }: { event: FullEvent; index: number })
     <Link to={to} unstable_viewTransition prefetch="intent">
       <div className="flex flex-col space-y-3 w-64 md:h-96 md:w-72 mx-6 sm:mx-0 max-w-md p-6 rounded-2xl bg-primary hover:transform hover:scale-105 transition-transform duration-300 hover:border">
         {event.imgUrl && event.imgAlt ? (
-          <img
+          <Image
             className={`${index === 1 && 'sm:order-2 pt-8'}  w-full object-contain box-border`}
             src={event.imgUrl}
             alt={event.imgAlt}
             style={{ viewTransitionName: isTransitioning ? 'image-expand' : '' }}
+            width={300}
+            height={200}
           />
         ) : (
-          <img
+          <Image
             className={`${index === 1 && 'sm:order-2 pt-8'}  w-full object-contain box-border`}
-            src={'https://res.cloudinary.com/dxctpvd8v/image/upload/v1709188683/default-event-photo_bvdslj.png'}
-            alt={'Default animated event zoom meeting'}
+            src={staticImage.defaultEventPhoto.url}
+            alt={staticImage.defaultEventPhoto.altText}
+            title={staticImage.defaultEventPhoto.title}
```

Comment:
Generally, using the title attribute is not recommended (see my research in Discord). However, it can be beneficial for SEO for the marketing pages.